[Optimize][Context] Execute RegisterCtxBuiltin in advance when pre-creating…

jianliang00 opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/lynx-family/lynx?pullRequest=982) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/lynx-family/lynx?pullRequest=982) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/lynx-family/lynx?pullRequest=982) it.</sub>

DragonDragonDDD committed:
[Optimize][Context] Execute RegisterCtxBuiltin in advance when pre-creating Context in QuickContextPool.

RegisterCtxBuiltin can be executed asynchronously during the pre-creation of QuickContext in QuickContextPool to reduce performance overhead on the main thread.

issue: m-6565536025
AutoSubmit: true
